packages/context/src/__tests__/unit/context-view.unit.ts

```diff
@@ -90,6 +90,59 @@ describe('ContextView', () => {
     expect(await taggedAsFoo.values()).to.eql(['BAR', 'FOO']);
   });
 
+  describe('EventEmitter', () => {
+    let events: string[] = [];
+
+    beforeEach(setupListeners);
+
+    it('emits close', () => {
+      taggedAsFoo.close();
+      expect(events).to.eql(['close']);
+      // 2nd close does not emit `close` as it's closed
+      taggedAsFoo.close();
+      expect(events).to.eql(['close']);
+    });
+
+    it('emits open', () => {
+      // At this point the view is opened and call `open` again won't emit
+      // `open` event
+      taggedAsFoo.open();
```

Comment:
Please add a check that no events were emitted, to make this test code easier to read & understand.

```ts
taggedAsFoo.open();
expect(events).to.eql([]);
```

Reply:
`open` is removed.